server/models/article.js

```diff
@@ -20,7 +39,7 @@ const articleSchema = new Schema({
     unique: true,
     required: true,
   },
-  title: { type: String, trim: true, required: true },
+  info: mercurySchema,
```

Comment:
Yeah, it's because I'm making the mercury info a subdocument of an article, not its own document (since there should be a 1-to-1 relationship between articles and info). I also disabled the _id field in the subdocument since we shouldn't have to access them separately.